utils.py

```diff
@@ -0,0 +1,108 @@
+from syslog import LOG_DEBUG
+
+import iso8601
+from pyparsing import Word, Suppress, nums, Optional, Regex, pyparsing_common, alphanums
+
+
+def parse_log_lines(body):
+    parser = Parser()
+    return [l for l in parser.parse_rfc6587(body) if filter_log_lines(l)]
+
+
+def filter_log_lines(msg):
```

Comment:
This works as an initial placeholder, but I think this will be the first piece to be reworked. Will probably want some not-hardcoded way to include filters, as there will be a lot.